default/stringtables/en.txt

```diff
@@ -8910,7 +8910,14 @@ Prefer [[PT_BARREN]] planets.
 
 SP_TRENCHERS_DESC
 '''Description:
-Coming soon.
+The Trenchers are an enigma of the galaxy. Who built them? Why do they constantly dig their spiral trenches? Theorys abound: "The trenches are a rudimentary form of communication" or "The patterns are of religious importance to the Trenchers". No one really knows.
+
+Homeworld:
+The Trenchers homeworld is a barren planet. It is mostly flat except for the elaborate trenches dug by its inhabitants, and a series of artificial hills constructed from the soil left over by the Trenchers digging. It is nearly devoid of plant life, with the exception of a local moss that clings to the side of the trenches.
+
+Society:
+Little is known about trencher society, they seem to show little interest in anything apart from digging trenchs and the construction of new workers. The famous ships captain and explorer Captain Bourke reports having witnessed groups of Trenchers gathering together and spinning on their wheels in what he described as "some kind of ritualistic dance". However no one has ever been able to confirm the existence of this behavior, and many experts have hypothesised that Captain Bourke was adversely effected by consuming the local flora.
```

Comment:
The description in terms of observation by an specific named outsider character seems out of place.

Reply:
I agree. Was unsure how to write it best, without the specific reference. Rewritten try this